### PR TITLE
Play project muted

### LINKED ISSUE #395: Scratch 2.0 behavior: ctrl+greenflag to run the project muted
We could also allow users to mute the project after it started, and vice versa.

### PR DESCRIPTION
**Pull request by TheColaber**
Resolves #395

### CODE REVIEW

**Comment by YOYITsM3M8** [comment]
> done... I expect not a single comment, just a quick merge...

hmmmm ok

File: addons/60fps/userscript.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default async function ({ addon, global, console }) {
     let mode = false;
     const flagListener = (e) => {
       if (altPressedRecently) {
+        e.cancelBubble = true;
```
Comment (RedGuy12) [inline addons/60fps/userscript.js:21]: why do you need this line?

Comment (TheColaber) [reply addons/60fps/userscript.js:21]: This line will not allow the project to restart when the flag is pressed.

Comment (RedGuy12) [reply addons/60fps/userscript.js:21]: oh i was just confused as it doesn't look like it's related to this addon

**Pull request merged**